package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could add 2 fields: [repository](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository) and [homepage](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#homepage)